editions/tw5.com/tiddlers/wikitext/Formatting in WikiText.tid

```diff
@@ -1,19 +1,34 @@
 caption: Formatting
 created: 20131205155959399
-modified: 20220513120211686
+modified: 20220726132504220
 tags: WikiText
 title: Formatting in WikiText
 type: text/vnd.tiddlywiki
 
+<style>
+.no-code-border > code {
+  border: initial;
+  background-color: initial;
+}
+</style>
 Available character formatting includes:
 
-* <code>&#96;backticks&#96;</code> for `code` (<<.icon $:/core/images/mono-line>>)
-** Alternatively, <code>&#96;&#96;double backticks allows &#96;embedded&#96; backticks&#96;&#96;</code><br><br>
-* `''bold''` for ''bold text'' (<<.icon $:/core/images/bold>>)<br><br>
-* `//italic//` for //italic text// (<<.icon $:/core/images/italic>>)<br><br>
-* `__underscore__` for __underscored text__ (<<.icon $:/core/images/underline>>)<br><br>
-* `^^superscript^^` for ^^superscripted^^ text (<<.icon $:/core/images/superscript>>)<br><br>
-* `,,subscript,,` for ,,subscripted,, text (<<.icon $:/core/images/subscript>>)<br><br>
-* `~~strikethrough~~` for ~~strikethrough~~ text (<<.icon $:/core/images/strikethrough>>)
+<div class="no-code-border">
+
+|Wikitext | Editor Button |HTML |Rendered Output |h
+|```backticks` ``are used for  for code | (<<.icon $:/core/images/mono-line>>) |`<code>backticks</code>` are used for code|`backticks` are used for  for code |
```

Comment:
IMO the output looks better with `` `backticks` ``, i.e symmetrical spaces around it. Also note the double "for" in both the first and the last table columns.

Reply:
My CSS settings got messed up. It should look like this: 

![grafik](https://user-images.githubusercontent.com/374655/181173044-bff1e344-61eb-450a-9ddb-66c17f8f7fe4.png)